pulp_rpm/app/models/content.py

```diff
@@ -0,0 +1,72 @@
+import tempfile
```

Comment:
hmm, maybe having this in modes/repository.py or models/signing_service.py will be better?

Reply:
I've followed pulpcore, which defines the base SigningService under content. But I agree, `models/signing_service.py` feels better (and its what pulp_deb does also).